addons/l10n_ae_pos/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     'name': 'United Arab Emirates - Point of Sale',
     'author': 'Odoo PS',
-    'category': 'Accounting/Localizations/Point of Sale',
+    'category': 'Sales/Point of Sale',
```

Comment:
That's my humble opinion, but the POS is not only Sale, and not only printing receipts and invoices (Accounting)
To me, it's more like POS is a module category on its own, and this is a `Point of Sale/Localizations` module that requires a specific `Accounting/Localizations` module.
But then again, this is completely arbitrary from me, and I'd hear a PO about this.